### ISSUE TITLE
Random logit interpolation seems not to be applied in AdaMatch 

### ISSUE THREAD

**Issue by Miracle2333** [microsoft/Semi-supervised-learning#17]
Hi there, as I went through the codes, I found that Random logit interpolation seems not to be applied in AdaMatch algorithm. 

In the paper, the logits from labeled data is randomly interpolated from some logits of the unlabeled data, as described in the paper.
![image](https://user-images.githubusercontent.com/27708595/191441613-a4f2dc28-9184-45c1-9fc7-362b03966e96.png)

However, in your implementation, there seems no such an interpolation perfromed on the logits of labeled data and they are the same with those in FixMatch. If the logit interpolation is implemented at other parts, please inform us.

**Comment by Hhhhhhao** [comment]
Hi, AdaMatch is proposed as a unified framework for both domain adaptation and semi-supervised learning. The Random Logit Interpolation is used to match the batch statistics of the source domain and both source and target domain, which might be useful for domain adaption, but not for semi-supervised learning, since we currently assume the same source and target domain.

**Comment by Miracle2333** [comment]
Make sense！The experiments of AdaMatch also proves that.
